docs/content/en/docs/getting-started/production-environment/snow-getstarted.md

```diff
@@ -0,0 +1,224 @@
+
+   ```bash
+   eksctl anywhere generate clusterconfig $CLUSTER_NAME --provider snow > eksa-mgmt-cluster.yaml
+   ```
+
+1. Optionally import images to private registry
+   * [Configure Private Registry on Snowball](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) shows an AWS sample of selecting and building a private registry in a Snowball Edge device.
+   * For air-gapped scenario, run the `import images` with `--input` and `--bundles` arguments pointing to the artifacts and bundle release files that pre-exist in the admin instance.
+   * Refer to the [Registry Mirror configuration]({{< relref "../../reference/clusterspec/optional/registrymirror" >}}) for more information about using private registry.
+
+   ```bash
+   eksctl anywhere import images \
+      --input /usr/lib/eks-a/artifacts/artifacts.tar.gz \
+      --bundles /usr/lib/eks-a/manifests/bundle-release.yaml \
+      --registry $PRIVATE_REGISTRY_ENDPOINT \
+      --insecure=true
+   ```
+
+1. Modify the cluster config (`eksa-mgmt-cluster.yaml`) as follows:
+   * Refer to the [Snow configuration]({{< relref "../../reference/clusterspec/snow" >}}) for information on configuring this cluster config for a Snow provider.
+   * Add [Optional]({{< relref "/docs/reference/clusterspec/optional/" >}}) configuration settings as needed.
+
+1. Set License Environment Variable
+
+   If you are creating a licensed cluster, set and export the license variable (see [License cluster]({{< relref "/docs/tasks/cluster/cluster-license" >}}) if you are licensing an existing cluster):
+
+   ```bash
+   export EKSA_LICENSE='my-license-here'
+   ```
+
+1. Configure Curated Packages
+
+   The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. To request a free trial, talk to your Amazon representative or connect with one [here](https://aws.amazon.com/contact-us/sales-support-eks/). Cluster creation will succeed if authentication is not set up, but some warnings may be generated. Detailed package configurations can be found [here]({{< relref "../../tasks/packages" >}}).
+
+   If you are going to use packages, set up authentication. These credentials should have [limited capabilities]({{< relref "../../tasks/packages/#setup-authentication-to-use-curated-packages" >}}):
+   ```bash
+   export EKSA_AWS_ACCESS_KEY_ID="your*access*id"
+   export EKSA_AWS_SECRET_ACCESS_KEY="your*secret*key"
+   export EKSA_AWS_REGION="us-west-2" 
+   ```
+
+1. Set Credential Environment Variables
+
+   Before you create the initial cluster, you will need to use the `credentials` and `ca-bundles` files that are in the admin instance, and export these environment variables for your AWS Snowball device credentials.
+Make sure you use single quotes around the values so that your shell does not interpret the values:
+   
+   ```bash
+   export EKSA_AWS_CREDENTIALS_FILE=/PATH/TO/CREDENTIALS/FILE
```

Comment:
I'd suggest adding single quotes around each value, as suggested in the text. If you like, you could put the paths in italic.